### PR TITLE
[SHELL32][PAINT][DRIVERS][SHELLEXT] French translation update

### PR DESCRIPTION
**Pull request by KyleKatarn31**
## Purpose

French Translation following
- https://github.com/reactos/reactos/pull/3056
- https://github.com/reactos/reactos/pull/3100

Previous updates on Paint
Missing FR translation in drivers

### CODE REVIEW

**Comment by binarymaster** [comment]
Cc @reactos/lang-french

**Comment by KyleKatarn31** [comment]
Sorry ColinFinck, Extravert-ir, HeisSpiter, ThFabba and tkreuzer for the undue review request (wrong Git command)

**Comment by KyleKatarn31** [comment]
@HBelusca @sdever would you please review and approve if a french native speaker is needed before go to merge ?

**Comment by KyleKatarn31** [comment]
@katahiromz @binarymaster : Do you need anything from me in order to Approve & merge ?
(like for https://github.com/reactos/reactos/pull/3047 even if RH does not apply here)

**Review by katahiromz — APPROVED** [review]

**Review by sdever — APPROVED** [review]

File: dll/shellext/stobject/lang/fr-FR.rc
Comment (HBelusca) [inline dll/shellext/stobject/lang/fr-FR.rc:22]:
```suggestion
    IDS_PWR_CHARGING          "%u%% en charge"
```

File: dll/shellext/stobject/lang/fr-FR.rc
Comment (HBelusca) [inline dll/shellext/stobject/lang/fr-FR.rc:23]:
```suggestion
    IDS_PWR_UNKNOWN_REMAINING "Restant inconnu"
```

File: dll/shellext/stobject/lang/fr-FR.rc
Comment (HBelusca) [inline dll/shellext/stobject/lang/fr-FR.rc:38]:
```suggestion
    IDS_HOTPLUG_CLICK "Avant de retirer ce périphérique, cliquez sur cette icône et sélectionnez le périphérique à retirer."
```

File: dll/win32/shell32/lang/fr-FR.rc
Comment (HBelusca) [inline dll/win32/shell32/lang/fr-FR.rc:1002]:
```suggestion
    IDS_COPYTOTITLE "Sélectionnez l'emplacement où vous souhaitez copier '%s', puis appuyez sur le bouton Copier."
```

File: dll/win32/shell32/lang/fr-FR.rc
Comment (HBelusca) [inline dll/win32/shell32/lang/fr-FR.rc:1006]:
```suggestion
    IDS_MOVETOTITLE "Sélectionnez l'emplacement où vous souhaitez déplacer '%s', puis appuyez sur le bouton Déplacer."
```

File: drivers/storage/class/cdrom/cdrom.inf
Comment (HBelusca) [inline drivers/storage/class/cdrom/cdrom.inf:66]:
```suggestion
```

File: dll/shellext/stobject/lang/fr-FR.rc
Comment (HBelusca) [inline dll/shellext/stobject/lang/fr-FR.rc:30]: Maybe s/matériel/périphérique/ (as done below)

Comment (KyleKatarn31) [reply dll/shellext/stobject/lang/fr-FR.rc:30]: I align with "Périphérique" in all places for the sake of consistency with your other comments.

**Comment by KyleKatarn31** [comment]
All comments from @HBelusca taken into account (thanks !)

File: drivers/storage/class/cdrom/cdrom.inf
Comment (ThFabba) [inline drivers/storage/class/cdrom/cdrom.inf:60]: Can you keep the sections sorted please? Ignore the '04', then it goes 08,09,0a,0b,0c, ...
And in doing that, you'd see there is already a .040c section.

File: drivers/storage/class/ramdisk/ramdisk.inf
Comment (ThFabba) [inline drivers/storage/class/ramdisk/ramdisk.inf:78]: This should go before 0415.

File: drivers/storage/class/disk/disk.inf
Comment (ThFabba) [inline drivers/storage/class/disk/disk.inf:69]: Same here, move down please.

File: drivers/storage/class/cdrom/cdrom.inf
Comment (KyleKatarn31) [reply drivers/storage/class/cdrom/cdrom.inf:60]: ok

File: drivers/storage/class/ramdisk/ramdisk.inf
Comment (KyleKatarn31) [reply drivers/storage/class/ramdisk/ramdisk.inf:78]: ok

File: drivers/storage/class/disk/disk.inf
Comment (KyleKatarn31) [reply drivers/storage/class/disk/disk.inf:69]: ok

**Comment by KyleKatarn31** [comment]
Comments from @ThFabba taken into account. Ready to go ?

**Comment by KyleKatarn31** [comment]
Would it be possible to merge now that no more comments are open ?

File: dll/win32/shell32/lang/fr-FR.rc
Comment (HBelusca) [inline dll/win32/shell32/lang/fr-FR.rc:1001]:
```suggestion
    IDS_COPYTOMENU "Copie&r dans un dossier..."
```

File: dll/win32/shell32/lang/fr-FR.rc
Comment (HBelusca) [inline dll/win32/shell32/lang/fr-FR.rc:1005]:
```suggestion
    IDS_MOVETOMENU "&Déplacer vers un dossier..."
```

**Review by HBelusca — APPROVED** [review]
Fix keyboard accelerators.

**Comment by KyleKatarn31** [comment]
Thank you @HBelusca

**Pull request merged**
